codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
```

Comment:
Shouldn't we use:

```yaml
coverage:
  status:
    patch:
      default:
        enabled: yes
        target: 100%
```

?

Reply:
Yes you're right, we should enforce 100% coverage of new code !